tutorials/pulse/3_building_pulse_schedules.ipynb

```diff
@@ -132,7 +132,7 @@
    "source": [
     "### `append` or `+`\n",
     "\n",
-    "The `append` method is like `insert`, but the insertion time is determined for us. The `Instruction` or `Schedule` being added will begin when all the channels common to the two become free. If they contain no common channels, then the `Schedule` will be appended at `time=0`. In psuedocode:\n",
+    "The `append` method is like `insert`, but the insertion time is determined for us. The `Instruction` or `Schedule` being added will begin when all the channels common to the two become free. If they contain no common channels, then the `Schedule` will be appended at `time=0`. In pseudocode:\n",
```

Comment:
Could we surround = with spaces, so "time = 0"? I think it contributes and relates to good code practices.

Reply:
Yes , it will look good.